etc/portage/sets/eessi-2022.11-linux-x86_64

```diff
@@ -0,0 +1,19 @@
+app-editors/emacs
+app-editors/vim
+app-shells/bash-completion
+dev-python/GitPython
+dev-python/keyring
+dev-python/pip
+dev-python/pycodestyle
+dev-python/pyyaml
+dev-python/rich
+dev-util/patchelf
+media-fonts/dejavu
+media-fonts/liberation-fonts
+=sys-apps/archspec-0.1.3::eessi
+sys-apps/dbus
+sys-cluster/lmod
+sys-cluster/rdma-core
+=sys-cluster/reframe-3.9.1::eessi
```

Comment:
@trz42 ReFrame 3.12.0 is the latest version, we should bump to that?

Or, we should remove ReFrame from the compat layer all together, see discussion in https://github.com/EESSI/software-layer/issues/127

Reply:
If we remove it, can we then still run tests while building the compat layer?

Reply:
Sure, we can (temporarily) install `reframe` when we need it (via `pip install`).

Reply:
Updated to ReFrame 3.12.0